portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+var $profile = $('#profile'),
+    lFollowX = 0,
+    lFollowY = 0,
+    lFollowZ = 0,
+    x = 0,
+    y = 0,
+    friction = 1 / 10;
+
+$(document).ready(function() {
+  function animate() {
+    x += (lFollowX - x) * friction;
+    y += (lFollowY - y) * friction;
+    $('#profile').css({
+        'transform': 'perspective(600px) rotateY(' + -x + 'deg) rotateX(' + y + 'deg)'
+    });
+    window.requestAnimationFrame(animate);
+  }
+
+  $("#profile").on('mousemove click', function(e) {
```

Comment:
Could this be a static function and then you bind the mouse event to this method in your html? I actually don't know what is the best practice. To bind events to methods in the js file or in the element definition in the html file?

Reply:
See the explanation above for an explanation as to why I bound events to methods in the js file. Thanks!